kubernetes/src/main/java/io/kubernetes/client/openapi/ApiClient.java

```diff
@@ -518,8 +518,9 @@ public boolean isDebugging() {
     public ApiClient setDebugging(boolean debugging) {
         if (debugging != this.debugging) {
             if (debugging) {
-                loggingInterceptor = new HttpLoggingInterceptor();
+                loggingInterceptor = new HttpLoggingInterceptor(debugLogger);
                 loggingInterceptor.setLevel(Level.BODY);
+                loggingInterceptor.redactHeader("authorization");
```

Comment:
is it case sensitive? can we add a test cases to see if `Authorization` is also redacted?

Reply:
sure. I think we are the ones adding this header and we add it as lowercase, but no problem regardless

Reply:
done